elytron/EAP7-1856-Bearer-Token-Authorization-Timeout.adoc

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+  - elytron
+---
+= [Preview] Bearer token timeout configurability will be added to WildFly's Elytron subsystem.
+:author:            Rebecca Searls
+:email:             rsearls@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:
+
+== Overview
+
+Bearer Token Authorization is the process of authorizing HTTP requests based on
+the existence and validity of a bearer token.  The token carries within it
+an expiration timestamp.  The two parameters being added, connection-timeout
+and read-timeout are placed on the URL used in retrieving the public key.
```

Comment:
"are placed on the URL" -> It would be good to reword this last sentence a bit to explain what the attributes will be used for, i.e., used when obtaining the public key from the OAuth provider.

Reply:
done